Html

kerrialison opened the pull request:
Added kerri varner's advice.

acodeinprogress commented:
Hey @kerrialison,

I noticed that you didn't add an advice page or any css and in fact, you deleted the main style.css file. If you could add that file back in as well as an advice page and some css, that would be great! Once you do that, you can add, commit, and push your work and it will join your pull request here.

Let us know if you need any help!

xoxo,
Sharon

kerrialison commented:
Yikes...I do have it locally - or had it! I'll take care of it now, sorry.

On Fri, Sep 7, 2018 at 12:57 PM, Sharon <notifications@github.com> wrote:

> Hey @kerrialison <https://github.com/kerrialison>,
>
> I noticed that you didn't add an advice page or any css and in fact, you
> deleted the main style.css file. If you could add that file back in as well
> as an advice page and some css, that would be great! Once you do that, you
> can add, commit, and push your work and it will join your pull request here.
>
> Let us know if you need any help!
>
> xoxo,
> Sharon
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/skillcrush-104/pull/2109#issuecomment-419502218>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AS_1H-LGGODmehrsLSBUYg-FG_Ojk9tWks5uYqWGgaJpZM4WbByv>
> .
>



-- 

instagram <https://www.instagram.com/kerri.alison/>
facebook <https://www.facebook.com/bleuinlove?ref=hl>

kerrialison commented:
I'm wondering if I somehow made a mistake in git because the advice.html
file is in my local repo as is style.css. I guess I do need help. Do I just
merge it with upstream again?

On Sat, Sep 8, 2018 at 8:42 AM, kerri varner <kerrivarner@gmail.com> wrote:

> Yikes...I do have it locally - or had it! I'll take care of it now, sorry.
>
> On Fri, Sep 7, 2018 at 12:57 PM, Sharon <notifications@github.com> wrote:
>
>> Hey @kerrialison <https://github.com/kerrialison>,
>>
>> I noticed that you didn't add an advice page or any css and in fact, you
>> deleted the main style.css file. If you could add that file back in as well
>> as an advice page and some css, that would be great! Once you do that, you
>> can add, commit, and push your work and it will join your pull request here.
>>
>> Let us know if you need any help!
>>
>> xoxo,
>> Sharon
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/skillcrush/skillcrush-104/pull/2109#issuecomment-419502218>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AS_1H-LGGODmehrsLSBUYg-FG_Ojk9tWks5uYqWGgaJpZM4WbByv>
>> .
>>
>
>
>
> --
>
> instagram <https://www.instagram.com/kerri.alison/>
> facebook <https://www.facebook.com/bleuinlove?ref=hl>
>
>


-- 

instagram <https://www.instagram.com/kerri.alison/>
facebook <https://www.facebook.com/bleuinlove?ref=hl>

kerrialison commented:
Sharon,

You meant that I deleted the link to style.css in my advice.html maybe?
I've added that back in. I didn't create my own css so I will do that. I'm
curious about some things as I thought I might just restart this project -

If I try to clone it again to a new directory, I get a couple of errors
(see attachment). The first time I tried to clone it, I realized I hadn't
forked it yet so I did that and tried again to clone the repo, but it still
won't clone without a passphrase for the SSH.
I have a few other related questions...

   - Why, if I'm in a brand new directory, is git showing me remote repos
   from other directories?
   - If I delete a local repo what happens?
   - the kerri-varner-advice.html file is in my local repo so how is it
   possible that you can't see it?

Thank for your help!

Kerri

On Sat, Sep 8, 2018 at 8:45 AM, kerri varner <kerrivarner@gmail.com> wrote:

> I'm wondering if I somehow made a mistake in git because the advice.html
> file is in my local repo as is style.css. I guess I do need help. Do I just
> merge it with upstream again?
>
> On Sat, Sep 8, 2018 at 8:42 AM, kerri varner <kerrivarner@gmail.com>
> wrote:
>
>> Yikes...I do have it locally - or had it! I'll take care of it now, sorry.
>>
>> On Fri, Sep 7, 2018 at 12:57 PM, Sharon <notifications@github.com> wrote:
>>
>>> Hey @kerrialison <https://github.com/kerrialison>,
>>>
>>> I noticed that you didn't add an advice page or any css and in fact, you
>>> deleted the main style.css file. If you could add that file back in as well
>>> as an advice page and some css, that would be great! Once you do that, you
>>> can add, commit, and push your work and it will join your pull request here.
>>>
>>> Let us know if you need any help!
>>>
>>> xoxo,
>>> Sharon
>>>
>>> —
>>> You are receiving this because you were mentioned.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/skillcrush/skillcrush-104/pull/2109#issuecomment-419502218>,
>>> or mute the thread
>>> <https://github.com/notifications/unsubscribe-auth/AS_1H-LGGODmehrsLSBUYg-FG_Ojk9tWks5uYqWGgaJpZM4WbByv>
>>> .
>>>
>>
>>
>>
>> --
>>
>> instagram <https://www.instagram.com/kerri.alison/>
>> facebook <https://www.facebook.com/bleuinlove?ref=hl>
>>
>>
>
>
> --
>
> instagram <https://www.instagram.com/kerri.alison/>
> facebook <https://www.facebook.com/bleuinlove?ref=hl>
>
>


-- 

instagram <https://www.instagram.com/kerri.alison/>
facebook <https://www.facebook.com/bleuinlove?ref=hl>

redrambles commented:
Hey @kerrialison,

We haven't heard from you in a while so I'm going to close this pull request. Feel free to submit a new one when you're ready!

Cheers,
Ann :)
